.gitignore

```diff
@@ -12,6 +12,7 @@ tests/.coverage
 # Test state / virtualenvs
 .tox
 .coverage
+cover
```

Comment:
This can be changed via `--cover-html-dir`.  Given we don't add this in our other projects I'd prefer to just keep it consistent.